notebooks/quality_control.ipynb

```diff
@@ -0,0 +1,103 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ddb44fb7-ff2e-425c-9b00-42de09dadee4",
+   "metadata": {},
+   "source": [
+    "# DataJoint Calcium Imaging Quality Metrics\n",
+    "\n",
+    "## 1. ScanQualityMetrics\n",
+    "\n",
+    "### 1.1 Frame Metrics\n",
+    "Frame metrics are quantitative measurements used to evaluate the consistency of imaging for each individual frame over time. There are several parameters to consider, such as the minimum intensity, mean intensity, maximum intensity, and contrast of the calcium imaging frames.\n",
+    "\n",
+    "\n",
+    "**min_intensity:** The minimum intensity is the lowest fluorescence intensity value recorded during the imaging session. It reflects the level of background noise and the sensitivity of the imaging system. Changes in minimum intensity could indicate various issues, such as changes in imaging conditions or technical issues with the imaging equipment or software. In either case, it may be necessary to adjust the imaging conditions or troubleshoot the equipment or software to identify and correct the problem.\n",
```

Comment:
more than likely, min level is set by the ADC bias and dark current since many pixels receive no photons. In this case, it does not reflect the sensitivity of the system.